persistence/src/test/scala/hmda/persistence/processing/LocalHmdaEventProcessorSpec.scala

```diff
@@ -71,13 +100,32 @@ class LocalHmdaEventProcessorSpec extends ActorSpec {
     "process validation completed from event stream" in {
       val msg = s"validation completed for submission $submissionId"
       checkEventStreamMessage(msg, ValidationCompleted(submissionId))
+      checkSubmissionStatus(Validated)
     }
   }
 
+  def checkSubmissionStatus(status: SubmissionStatus): Assertion = {
+    //TODO: find a way to avoid having to sleep here
+    Thread.sleep(500)
```

Comment:
I'm pretty sure there is a way, but I haven't thought about it in detail. As you say, we're using it across the board